testutil/compose/compose/main.go

```diff
@@ -77,3 +93,7 @@ func newDefineCmd() *cobra.Command {
 
 	return cmd
 }
+
+func addDirFlag(cmd *cobra.Command) *string {
```

Comment:
why passing `*cobra.Command` and not `*pflag.FlagSet` ?

Reply:
yeah, cool, can do that as well.